cli/pcluster/constants.py

```diff
@@ -12,3 +12,5 @@
 PCLUSTER_STACK_PREFIX = "parallelcluster-"
 PCLUSTER_ISSUES_LINK = "https://github.com/aws/aws-parallelcluster/issues"
 CIDR_ALL_IPS = "0.0.0.0/0"
+DEFAULT_ARCH = "x86_64"
```

Comment:
Here I'd use DEFAULT_OS_ARCHITECTURE it's more self-explaining.

I'd use Architecture in place of Arch everywhere.

Reply:
I'm fine with using the full word rather than the abbreviation. I don't think OS should go in the name though, since OS and architecture are independent concepts.